packages/appstore/src/components/containers/trading-app-card-actions.tsx

```diff
@@ -27,11 +27,12 @@ const TradingAppCardActions = ({
     is_account_being_created,
     is_buttons_disabled,
     is_real,
+    as_disabled,
 }: Actions) => {
     switch (action_type) {
         case 'get':
             return (
-                <Button disabled={is_account_being_created} primary_light onClick={() => onAction?.()}>
+                <Button disabled={is_account_being_created} primary_light onClick={onAction} as_disabled={as_disabled}>
```

Comment:
The Button component does not have any `as_disabled` prop in it 🤔 
BTW you can use `is_buttons_disabled` and pass false to it from it's parent
```suggestion
                <Button disabled={is_account_being_created ||  is_buttons_disabled} primary_light onClick={onAction} >
```